charts/seed-monitoring/charts/prometheus/rules/apiserver-connectivity-check.rules.yaml

```diff
@@ -0,0 +1,13 @@
+groups:
+- name: apiserver-connectivity-check.rules
+  rules:
+  - alert: KubernetesServiceUnreachable
```

Comment:
Can we rename the alert e. g. `ApiServerUnreachableKubernetesService`?

Reply:
done